data/tests/fwupdtool.sh

```diff
@@ -45,6 +51,41 @@ echo "Showing hwids"
 run hwids
 rc=$?; if [ $rc != 0 ]; then error $rc; fi
 
+# ---
+echo "Creating Boot0001"
+run efivar-boot create 0001 Fedora shimx64.efi /tmp/fwupd/mnt
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+# ---
+echo "Setting BootOrder"
+run efivar-boot order 0001
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+# ---
+echo "Setting BootNext"
+run efivar-boot next 0001
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+# ---
+echo "Showing EFI boot"
+run efivar-boot
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+# ---
+echo "Showing EFI files"
+run efivar-files
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
+
+# ---
+echo "Deleting Boot0001"
+run efivar-boot delete 0001
+rc=$?; if [ $rc != 0 ]; then error $rc; fi
```

Comment:
another test should be added for deleting a non-existent entry (delete it twice)

Reply:
Done, good catch